build-system/test-configs/forbidden-terms.js

```diff
@@ -54,15 +55,20 @@ let ForbiddenTermDef;
  * @const {Object<string, string|!ForbiddenTermDef>}
  */
 const forbiddenTerms = {
-  'DO NOT SUBMIT': '',
+  'DO NOT SUBMIT': {
+    checkComments: true,
```

Comment:
This creates a duplicate warning.

![image](https://user-images.githubusercontent.com/254946/112509398-a8dd3680-8d4d-11eb-8c4b-11e43bacffc3.png)

I'd like this term to apply to `.css` and `.go` files as well, so I think we should remove/replace eslint rule `no-warning-comments` instead.

Reply:
Agree.